src/screens/SendCoinsConfirmScreen.tsx

```diff
@@ -58,19 +58,24 @@ export class SendCoinsConfirmScreen extends Component<Props> {
     const balance = fromWallet.balance;
     const incomingBalance = fromWallet.incoming_balance;
     const amount = this.getAmountByTx(txDecoded);
-
     if (isAlert) {
       return {
         availableBalance: satoshiToBtc(balance - amount.my - amount.foreign) - fee,
         pendingBalance: satoshiToBtc(incomingBalance + amount.my) - fee,
       };
     }
 
-    const decreasePending = pendingAmountDecrease ? roundBtcToSatoshis(pendingAmountDecrease) : 0;
+    if (pendingAmountDecrease !== undefined) {
+      const decreasePending = roundBtcToSatoshis(pendingAmountDecrease);
+      return {
+        availableBalance: satoshiToBtc(balance + amount.my),
+        pendingBalance: satoshiToBtc(incomingBalance).toNumber() - decreasePending,
+      };
+    }
```

Comment:
What does it fix? Seems to be unnecessary if statement.

Reply:
available balance is counted in different way